src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Callout/BitCallout.scss

```
@@ -1,4 +1,5 @@
 ﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/media-queries.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)